examples/authz/components/authz/GrantModal.tsx

```diff
@@ -179,8 +181,9 @@ export const GrantModal = ({ isOpen, onClose, chainName }: GrantModalProps) => {
                   onClick={() => {}}
                 />
               </PopoverTrigger>
+              <Box></Box>
```

Comment:
what is this box for?

Reply:
It's probably a mistake write, I 'll fixed it